### DISCUSSION TITLE
[Feature] Creative Commons License

### DISCUSSION THREAD

**Discussion by dmaggi** [immich-app/immich#8365] [Feature Request]
### The feature

The proposed feature involves incorporating a [Creative Commons](https://creativecommons.org/) licensing option for added image protection. This feature could be configured at either the album level or within the global configuration settings.
**With the advent of AI, there's a growing risk of our photos being used indiscriminately without our consent**. By integrating Creative Commons licensing, users gain the ability to assert their ownership rights and specify how their photos can be used by others.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Albums allow you to enter a description in which you can state any license or other info.

**Reply by dmaggi** [reply]
It's not the same have this information as part of metadata. If I share an album, and someone download some image, these will not have the metadata license embed. I think at least thumbnails created should have this information.

**Reply by mertalev** [reply]
I think automatically adding a license to images would be error-prone. But I do agree that it's valid to want shared images to have embedded licenses.

I haven't tested this, but I think we might even be stripping any embedded license when we generate the thumbnail and preview images. If we preserve this info, it would enable workflows where a license is added to an image before importing it to immich.

A later enhancement may be to allow the user to add a license through the UI and reflect this in the generated images.

**Reply by dmaggi** [reply]
Can we reopen the feat? as we mention is not the same have this information in the description, It's actually useless :/

**Comment by MastaBaba** [comment]
I also would like this feature. Embedded in the image metadata would be great, as well as displaying the licensing terms with the image inside the web-based interface, specifically relevant for images that are publicly available.

**Reply by dmaggi** [reply]
Yes, exactly!
